src/clj/game/cards/resources.clj

```diff
@@ -2607,6 +2625,37 @@
                 :cost [:trash]
                 :effect (effect (pump target 2 :end-of-turn))}]})
 
+(defcard "The Masque A"
+  {:abilities [{:cost [:click 1, :trash]
+                :label "Make a run and gain [click]. If successful, draw 1 card."
+                :prompt "Choose a server"
+                :choices (req runnable-servers)
+                :msg (msg "make a run on " target " and gain [click]")
+                :async true
+                :effect (effect
+                          (gain :runner :click 1)
+                          (register-events
+                            card
+                            [{:event :successful-run
+                              :unregister-once-resolved true
+                              :duration :end-of-run
+                              :async true
+                              :effect (effect (system-msg :runner "uses The Masque A to draw 1 card")
+                                              (draw :runner eid 1 nil))}])
+                          (make-run eid target nil card))}]})
+
+(defcard "The Masque B"
+  {:implementation "Successful run condition not implemented"
```

Comment:
Check out the event Mobius or Data Breach to see an easy method for implementing multi-run abilities.